### PR TITLE
fix(taskframework):  calculate capacity of task in  single node by node memory

### PR DESCRIPTION
**Pull request by krihy**
#### What type of PR is this?

 calculate capacity of task size in  single node by node memory

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by guowl3 — APPROVED** [review]
I think the best solution is to obtain the specifications of the expansion node through the API.
ok, i will discuss with other team

File: server/odc-server/src/main/resources/data.sql [hunk 1]
```diff
@@ -753,6 +753,8 @@ INSERT INTO config_system_configuration ( `key`, `value`, `description` ) VALUES
 INSERT INTO config_system_configuration ( `key`, `value`, `description` ) VALUES( 'odc.task-framework.k8s-properties.limit-cpu', '2', 'k8s pod limit cpu' ) ON DUPLICATE KEY UPDATE `id` = `id`;
 INSERT INTO config_system_configuration ( `key`, `value`, `description` ) VALUES( 'odc.task-framework.k8s-properties.limit-mem', '4096', 'k8s pod limit memory,unit is MB' ) ON DUPLICATE KEY UPDATE `id` = `id`;
 INSERT INTO config_system_configuration ( `key`, `value`, `description` ) VALUES( 'odc.task-framework.k8s-properties.max-node-count', '', 'k8s max node count' ) ON DUPLICATE KEY UPDATE `id` = `id`;
+INSERT INTO config_system_configuration ( `key`, `value`, `description` ) VALUES( 'odc.task-framework.k8s-properties.node-cpu', '', 'k8s node cpu' ) ON DUPLICATE KEY UPDATE `id` = `id`;
```
Comment (yizhouxw) [inline server/odc-server/src/main/resources/data.sql:756]: description missed

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/PodConfig.java [hunk 2]
```diff
@@ -59,4 +59,8 @@ public class PodConfig {
     private Long mountDiskSize;
 
     private Long maxNodeCount;
+
+    private Long nodeCpu;
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/PodConfig.java:63]: may double, for example cpu can be 0.5 core

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/PodConfig.java [hunk 3]
```diff
@@ -59,4 +59,8 @@ public class PodConfig {
...
+
+    private Long nodeMem;
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/PodConfig.java:65]: if hardcode unit, filed name may end with unit name.

better follow k8s configuration design.

File: server/odc-server/src/main/resources/data.sql (hunk 1 above)
Comment (krihy) [reply server/odc-server/src/main/resources/data.sql]: okey, done

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/PodConfig.java (hunk 2 above)
Comment (krihy) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/PodConfig.java]: done

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/PodConfig.java (hunk 3 above)
Comment (krihy) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/PodConfig.java]: okey, i rename to `nodeMemInMb` which follow other field

**Review by yizhouxw — APPROVED** [review]

**Pull request merged**